pkg/storage/cachedb/cachedb_driver.go

```diff
@@ -0,0 +1,18 @@
+package cachedb
```

Comment:
As an interface, would have preferred this to be 
pkg/storage/cache.go

Reply:
Moved to said location